environments/se/core/src/main/java/org/jboss/weld/environment/se/Weld.java

```diff
@@ -176,6 +176,14 @@
 @Vetoed
 public class Weld extends SeContainerInitializer implements ContainerInstanceFactory {
 
+    /**
+     * By default, the set of bean-defining annotations is fixed. If set to a {@link Set} of annotation classes, the set of bean-defining annotations is
+     * augmented with the contents of the {@link Set}.
```

Comment:
Perhaps we should copy over the limitations mentioned on the other method, as those will still apply.
See this - https://github.com/weld/core/blob/master/environments/se/core/src/main/java/org/jboss/weld/environment/se/Weld.java#L896-L897